fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/FedoraTypesUtils.java

```diff
@@ -145,6 +145,11 @@
      */
     public static Predicate<Node> isAcl = new AnyTypesPredicate(FEDORA_WEBAC_ACL);
 
+    /**
+     * Predicate for determining whether this {@link Node} is a TimeMap.
+     */
+    public static Predicate<Node> isTimeMap = new AnyTypesPredicate(FEDORA_TIME_MAP);
```

Comment:
If you agree with my previous comment about overriding the isOriginalResource method, then we may not need this.